hooks.yaml

```diff
@@ -14,6 +14,14 @@
   files: (\.tf)$
   exclude: \.terraform\/.*$
 
+- id: terraform_readme
+  name: Terraform Readme
+  description: Creates readme for terraform modules.
```

Comment:
description: Creates a README for Terraform modules using `terraform_config_inspect`.